packages/desktopjs/src/Default/default.ts

```diff
@@ -209,12 +209,8 @@ export namespace Default {
                         continue;
                     }
 
```

Comment:
Removed the manual check as browser does it for you.  Added support to specify targetOrigin on published messages in case you want to target different origin which previously was not supported.